cirq-google/cirq_google/engine/engine_program.py

```diff
@@ -69,8 +69,8 @@ async def run_sweep_async(
         description: Optional[str] = None,
         labels: Optional[Dict[str, str]] = None,
         *,
-        run_name: str = "",
-        device_config_name: str = "",
+        run_name: str,
+        device_config_name: str,
```

Comment:
NIT - please clean up redundant `If specified, `processor_id` is required to be set.` from the docstrings for `run_name` and `device_config_name` arguments here and in `run_async` docstring below.  Also please make sure the order of arguments in these docstrings is the same as in function signatures (ie, `processor_id` should be described first if it is the first required argument).

Reply:
Done.